04homework/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Homework4
+{
+    using System;
+
+    class Homework4
+    {
+        static int MaxValue(int x, int y)
+        {
+            return Math.Max(x, y);
+        }
+
+        static int MaxValue(params int[] numbers)
```

Comment:
it's ok =) maybe early for this time use it

Reply:
yes, it's ok to use it, but please use in homework only stuff we've learnt during the lesson, it's easy yo do with LINQ and with optional params